.devcontainer/Dockerfile

```diff
@@ -98,6 +97,12 @@ RUN echo "Install general purpose packages" && \
 
 RUN DEBIAN_FRONTEND=noninteractive apt-get install -y clangd-12
 
+# Install golang 1.18
+WORKDIR /usr/local
+RUN curl https://artifactory.magmacore.org/artifactory/generic/go1.18.linux-amd64.tar.gz --remote-name --location && \
+    tar -xzf go1.18.linux-amd64.tar.gz && \
+    cp -r go/bin/* /usr/local/bin/
+
```

Comment:
We should remove the "extraction directory" after use here to keep the image small, imho.

Reply:
Closing as same as https://github.com/magma/magma/pull/12424#discussion_r859978511.